src/picongpu/include/fields/FieldTmp.kernel

```diff
@@ -75,7 +75,10 @@ namespace picongpu
             auto cachedVal = CachedBox::create < 0, typename TmpBox::ValueType > ( BlockDescription_( ) );
             Set<typename TmpBox::ValueType > set( float_X( 0.0 ) );
 
-            ThreadCollective<BlockDescription_> collective( linearThreadIdx );
+            ThreadCollective<
+                BlockDescription_,
+                PMacc::math::CT::volume< typename BlockDescription_::SuperCellSize >::type::value
+            > collective( linearThreadIdx );
```

Comment:
Here it is just `linearThreadIdx` again!
![But why?](https://media.giphy.com/media/1M9fmo1WAFVK0/giphy.gif)